golem-common/src/serialization.rs

```diff
@@ -119,7 +118,7 @@ mod tests {
     #[test]
     pub fn roundtrip_component_id() {
         let example = Some(ComponentId::new_v4());
-        println!("example: {example:?}");
+        info!("example: {example:?}");
```

Comment:
Should we have it at all? If there is a bug, someone can put `info`? Or we have a standard that some complex tests have logs?